packages/pluggableWidgets/maps-web/src/Maps.editorPreview.tsx

```diff
@@ -1,23 +1,22 @@
-import { createElement, ReactNode, Fragment } from "react";
-import { MapSwitcher } from "./components/MapSwitcher";
-
+import { createElement, ReactNode, Fragment, ReactElement } from "react";
 import { MapsPreviewProps } from "../typings/MapsProps";
-import { Marker } from "../typings/shared";
 import { Alert } from "@mendix/piw-utils-internal/components/web";
-import { parseStyle } from "@mendix/piw-utils-internal";
-import { translateZoom } from "./utils/zoom";
+import GoogleMapSvg from "./assets/GoogleMapsDesign.svg";
 
 declare function require(name: string): string;
 
 export const preview = (props: MapsPreviewProps): ReactNode => {
-    const locations: Marker[] = [
-        {
-            latitude: 51.906688,
-            longitude: 4.48837,
-            title: "Mendix Office Rotterdam",
-            url: ""
-        }
-    ];
+    const renderImagePreview = (): ReactElement => {
```

Comment:
As it does not depend on any external variable/inputs it could be a simple `const` with static value instead of a function.